types/react/test/tsx.tsx

```diff
@@ -41,6 +41,16 @@ StatelessComponent2.defaultProps = {
     <b>foo</b>
 </div>;
 
+// WAI-ARIA 1.1 Attributes
+<div
+    aria-atomic={false}
+    aria-checked='true'
+    aria-colcount={7}
+    aria-label='test'
```

Comment:
Is there any way to have a test for a failure case?

As far as I understand, the typescript compiler just has a blanket whitelist for any property with a `-` in its name. I don't know if it will type-check the property if a type declaration exists.

At any rate, this _is_ useful, even if not checked, for IDE code assists and documentation.

Reply:
I wasn't able to find, but if there is a way, I'd love to know!

Reply:
You can add `// $ExpectError` to a line to assert that there is an error on that line.